queues/priority_queue.py

```diff
@@ -0,0 +1,41 @@
+"""
+Implementation of priority queue
+"""
+
+class PriorityQueueNode:	
+	def __init__(self, data, priority):
+		self.data = data
+		self.priority = priority
+
+	def __repr__(self):
+		return str(self.data) + ": " + str(self.priority)
+
+class PriorityQueue:
+	def __init__(self):
+		self.priority_queue_list = []
+
+	def size(self):
+		return len(self.priority_queue_list)
+
+	def insert(self, node):
```

Comment:
And yeah, there is no point of returning `True` here, because it would return `True` _every time_, and there is no case for `False`. So, its better to return `None`. 
Returning `True` and `False` is more logical if there was, for example, a maxsize restriction in the queue. Then we would have returned `True` for every successful insertion, while returned `False` if the queue was already full :)